build/images/Dockerfile.build.agent.ubuntu

```diff
@@ -29,6 +29,9 @@ RUN --mount=type=cache,target=/go/pkg/mod/ \
     --mount=type=cache,target=/root/.cache/go-build/ \
     make antctl-linux && mv bin/antctl-linux bin/antctl
 
+
+RUN apt-get update && apt-get install -y libpcap-dev && apt-get autoremove -y && rm -rf /var/cache/apt/* /var/lib/apt/lists/*
```

Comment:
yeah i'm not totally sure about introduce this libpcap and cgo thing because it required lots of changes in the build files. will move this part once it's confirmed.